src/Data/Formatter/DateTime.purs

```diff
@@ -201,10 +209,14 @@ formatF cb dt@(DT.DateTime d t) = case _ of
     (if (fromEnum $ T.hour t) >= 12 then "PM" else "AM") <> cb a
   Minutes a →
     show (fromEnum $ T.minute t) <> cb a
+  MinutesTwoDigits a →
+    (padSingleDigit <<< fromEnum $ T.minute t) <> cb a
   Seconds a →
     show (fromEnum $ T.second t) <> cb a
+  SecondsTwoDigits a →
+    (padSingleDigit <<< fromEnum $ T.second t) <> cb a
   Milliseconds a →
-    show (fromEnum $ T.millisecond t) <> cb a
```

Comment:
I don't know whether switching to `padSingleDigit` here is right or not - I don't think it's quite right though. I would have though maybe we need to pad it out to 3 digits (000-999) or not at all, so maybe this is another variety of option that needs adding too?